server/src/main/java/com/hedera/block/server/notifier/NotifierConfig.java

```diff
@@ -38,14 +39,8 @@ public record NotifierConfig(@ConfigProperty(defaultValue = "1024") int ringBuff
      * @throws IllegalArgumentException if the configuration is invalid
      */
     public NotifierConfig {
-        if (ringBufferSize <= 0) {
-            throw new IllegalArgumentException("Ring buffer size must be greater than 0");
-        }
-
-        if ((ringBufferSize & (ringBufferSize - 1)) != 0) {
-            throw new IllegalArgumentException("Ring buffer size must be a power of 2");
-        }
-
+        Preconditions.requirePositive(ringBufferSize, "Notifier Ring Buffer Size must be positive!");
+        Preconditions.requirePowerOfTwo(ringBufferSize, "Notifier Ring Buffer Size must be a power of 2!");
```

Comment:
Remove exclamation points

Reply:
✅ Done. Do we have some convention for the error messages? If so, let me know. Thanks!